english/beach/beach.md

```diff
@@ -0,0 +1,5 @@
+You know in your heart the beach is where you wanna be. The sun, the sand, the sea and the breeze. You close your eyes and you can just imagine the beauty. Without any hesitation you head for the door.
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.